docs/src/Guides/01 Getting Started.md

```diff
@@ -9,8 +9,30 @@ Hi! So you want to make a bot powered by snakes. This guide aims to get you star
 - [x] [A bot account](02 Creating Your Bot.md)
 - [ ] An aversion to snakes
 
-## Installation
+## Using a Template
 
+We created a [cookiecutter template](https://github.com/Discord-Snake-Pit/Bot-Template) which you can use to set up your own bot faster.
+With the template, your code will already have a well-defined structure which will make development easier for you.
+
+We recommend newer devs to make use of this template.
```

Comment:
Really? Most new devs have no idea what cookiecutter is

Reply:
Sure, but it's not complicated to use. I'd recommend them using it because:
1) They don't have to deal with setting up the same basic structure for every project (more for exp devs)
2) They learn about some of the very useful features we have - eg. scales
3) They may need less help setting everything up - better for us
4) They may not fall into bad habits such as putting tokens in their files, or putting all code in one file